spek-dsl/common/src/main/kotlin/org/spekframework/spek2/dsl/SpecBody.kt

```diff
@@ -25,4 +25,92 @@ interface SpecBody: TestContainer {
 
     fun beforeGroup(callback: () -> Unit)
     fun afterGroup(callback: () -> Unit)
+
+    /**
+     * Creates a [group][SpecBody.group].
+     *
+     * @author Ranie Jade Ramiso
+     * @since 1.0
+     */
+    @Synonym(type = SynonymType.Group, prefix = "describe")
+    fun describe(description: String, body: SpecBody.() -> Unit) {
+        group("describe $description", body = body)
+    }
+
+    /**
+     * Creates a [group][SpecBody.group].
+     *
+     * @author Ranie Jade Ramiso
+     * @since 1.0
```

Comment:
Maybe remove these?

Reply:
I would remove that, but it's @raniejade's child, so only if he agrees :)

Reply:
:+1: actually I just commented about it :).